docs/01-overview/main-areas/observability/README.md

```diff
@@ -2,20 +2,32 @@
 title: What is Observability in Kyma?
 ---
 
-## Observability tools
+Modern cloud applications consist of multiple components that can be scaled and deployed individually, and are focused on one concern using the most fitting technology - essentially, they're distributed systems. Monitoring such an application requires re-thinking: It's no longer an action you apply to a specific component, instead applications must actively expose insights! Monitoring becomes a federated aspect for all components, where every single component must actively expose its internal state.
```

Comment:
This paragraph explains the motivation/vision behind the Kyma Observability concept. It's not strictly technical but considering the README is the first point of contact with the topic, I think it could be justified.

Reply:
I think it's still valuable to keep the change of "direction" - metrics aren't "pulled" anymore, but actively "pushed" by the component. I'll propose a compromise between your suggestion and the earlier version.

Reply:
removed with https://github.com/kyma-project/kyma/pull/13119/commits/4e6932d2eff1afc834ed9a341ce6f0717e952f22